### ISSUE TITLE
NTR: Hyaluronidase low in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8215]
New term request Hyaluronidase low in blood (segal_100329190414):

**Comment by pnrobinson** [comment]
@MickeySegal
Is the measurement of hyaluronidase activity done for each of the three enzymes separately or is this a combined test? What is the disease context?

**Comment by MickeySegal** [comment]
This is from https://www.nejm.org/doi/full/10.1056/NEJM199610033351405
OMIM: https://omim.org/entry/601492

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000969
- label: Diminished circulating hyaluronidase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:8793927
- def: The activity or concentration of hyaluronidase in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8215
- synonyms:
